.github/workflows/test.yaml

```diff
@@ -120,12 +120,17 @@ jobs:
       - uses: actions/checkout@v2
       - name: Verify wandb server is running
         run: curl -s http://wandbservice:8080/healthz
-      - name: Run tests
+      - name: Run Python Unit Tests (Clickhouse Client Only)
+        env:
+          DD_SERVICE: weave-python
+          DD_ENV: ci
+          WEAVE_SENTRY_ENV: ci
+        run: CI=1 WB_SERVER_HOST=http://wandbservice WF_CLICKHOUSE_HOST=weave_clickhouse WEAVE_SERVER_DISABLE_ECOSYSTEM=1 source /root/venv/bin/activate && cd weave && pytest -m "weave_client and not vcr" --weave-server=clickhouse --job-num=${{ matrix.job_num }} --timeout=90  ./flow ./integrations ./tests ./ops_arrow ./ecosystem ./trace_server ./trace --ddtrace --durations=5
```

Comment:
`and not vcr` is included here since i could not get the vcr module to play nicely with out clickhouse instance